language/cc/config.go

```diff
@@ -270,6 +282,8 @@ func newCcConfig() *ccConfig {
 		generateCC:              true,
 		generateProto:           true,
 		platforms:               map[platform.Platform]platformConfig{},
+		ccIncludePrefix:         "",
+		ccStripIncludePrefix:    "",
```

Comment:
No need to initialize these. Go initializes variables to the zero value by default, which is the empty string here.